test/browser/main_test.js

```diff
@@ -41,7 +41,7 @@ describe('main process', () => {
 
   describe('when required', () => {
     before(
-      async () => { await import('../../src/browser/main') }
+      () => { import('../../src/browser/main') }
```

Comment:
Careful, you need to use either `() => import()` or `() => (import())` or `() => { return import() }` -- otherwise the promise will not be returned.